host/greatfet/protocol/vendor_requests.py

```diff
@@ -26,7 +26,6 @@
 
     # Temporary, custom stuffs?
     'ENABLE_USB1',
-    'SET_LEDS',
```

Comment:
Don't remove this line, it changes the index of all the ones below it.

Reply:
Fixed.